pkg/ko/build/gobuild_test.go

```diff
@@ -31,22 +31,26 @@ type testContext struct {
 	workdir string
 }
 
-func (tc *testContext) Enter(t *testing.T) {
+func TestMain(m *testing.M) {
 	// Track the original state, so that we can restore it.
 	ogp := os.Getenv("GOPATH")
+
+	// Run the tests.
+	code := m.Run()
+
+	// Restore everything.
+	getwd = os.Getwd
+	os.Setenv("GOPATH", ogp)
+
+	os.Exit(code)
```

Comment:
Are you sure this is still necessary?  EnvVars should be scoped to the process.

Reply:
Tests passed without it, gone!